### PR TITLE
Set TEST_JDK_HOME by environment variable

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophiag@ca.ibm.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Test: https://github.com/sophia-guo/runaqaTest/actions/runs/59166067
https://github.com/sophia-guo/runaqaTest/actions/runs/59165200
https://github.com/sophia-guo/runaqaTest/actions/runs/59164467


Depends on https://github.com/AdoptOpenJDK/install-jdk/pull/2

File: action.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@ name: 'Running AQA test'
...
 author: 'Sophia Guo'
 inputs:
-  version:              # jdk version
-    description: 'testing jdk version'
-    default: '8'
   jdksource:               # customized: if installed jdk by TheMrMilchmann/install-jdk or default: github hosted environment  
     description: 'git-hub hosted default or customized'
     required: false
+  version:              # jdk version
+    description: 'testing jdk version (Required when jdksource is default git-hub hosted.)'
```
Comment (smlambert) [inline action.yml]: Do not need a - in github (link 6 and 9)
You will want to update line 8 as well to point to AdoptOpenJDK/install-jdk

otherwise LGTM

**Comment by smlambert** [comment]
@sophia-guo - I forgot where we left off on this one, are we good to proceed with this PR?

**Comment by sophia-guo** [comment]
Sorry I should close this one as TEST_JDK_HOME won't be setup as environment variable in install-jdk action.

**Pull request closed without merging**